hack/e2e.sh

```diff
@@ -52,5 +52,6 @@ while [ $ATTEMPTS -ge 0 ]; do
   fi
 done
 
-# Check that policy engine returns channel data respond
-curl --header 'Accept:application/json' "${GRAPH_URL}"
+# Run e2e tests
+cd ../../graph-builder
+cargo test --features test-e2e e2e --no-default-features  -- --nocapture
```

Comment:
This would install test dependencies during the test run, which is suboptimal. I'll rework this in /release repo to ensure a test image is built first

Reply:
I'm not sure what you mean by "install test dependencies", please explain.

Reply:
This test starts with a container built from `Dockerfile.builder`, so it has rustc and cargo installed, but doesn't have test prebuilt. So when `cargo test` is run here it would download crates and compile test - [log](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cincinnati/202/pull-ci-openshift-cincinnati-master-e2e/23/artifacts/e2e/container-logs/test.log)

I'll update this to have test built along with other artifacts in follow up PRs